apps/trivium/src/trivium/test.rs

```diff
@@ -0,0 +1,411 @@
+use tfhe::prelude::*;
+use tfhe::{generate_keys, ConfigBuilder, FheBool, FheUint64, FheUint8};
+
+use crate::{TransCiphering, TriviumStream, TriviumStreamByte, TriviumStreamShortint};
+
+///! Values for these tests come from: https://github.com/cantora/avr-crypto-lib/blob/master/testvectors/trivium-80.80.test-vectors
```

Comment:
I think ///! is not a doc comment you would need to use //! maybe at the start of the file, can you maybe select a specific commit from that repo if the authors make updates and move stuff (unlikely I agree) it would break this link